Cargo.toml

```diff
@@ -620,6 +620,11 @@ name = "isaac"
 path = "src/isaac.rs"
 
 [[bin]]
+# http://rosettacode.org/wiki/Jensen's_Device
+name = "jensens_device"
+spath = "src/jensens_device.rs"
```

Comment:
It looks like there is an extra `s` at the beginning of the line that shouldn't be there